pages/linux/repquota.md

```diff
@@ -0,0 +1,27 @@
+# repquota
+
+> Display a summary of existing file quotas for a filesystem.
+
+- Report on all existing quotas:
+
+`repquota -all`
+
+- Report all quotas, even if there is no usage:
+
+`repquota -v`
+
+- Report on quotas for users (default), only super-user may view quotas which are not their own:
```

Comment:
When I say default, I mean running sudo repquota {{user}} will do the same thing as sudo repquota -u {{user}}. 
Okay, how about having two entries, one for reporting quota on your own user, and one for reporting quota on other users (as root) - 

- Report on quotas for current user:

`repquota --user {{current_user}}`

- Report on quotas for another user:

`sudo repquota --user {{another_user}}`

- Report on quotas for user groups:

`sudo repquota --group {{group}}`

I also added sudo for group. 

The command itself is generally an administrators tool, only the super-user may view quotas which are not their own. Running without sudo generates a permissions error. 
Should I add sudo to all of the commands?